processing/src/test/java/org/apache/druid/query/groupby/GroupByQueryRunnerTest.java

```diff
@@ -449,7 +451,9 @@ public static Collection<Object[]> constructorFeeder()
           final String testName = StringUtils.format("config=%s, runner=%s, vectorize=%s", config, runner, vectorize);
 
           // Add vectorization tests for any indexes that support it.
-          if (!vectorize || QueryRunnerTestHelper.isTestRunnerVectorizable(runner)) {
+          if (!vectorize ||
+              (QueryRunnerTestHelper.isTestRunnerVectorizable(runner) &&
+               config.getDefaultStrategy().equals(GroupByStrategySelector.STRATEGY_V2))) {
```

Comment:
why was this change needed?

Reply:
this was needed because strategy-v1 doesn't implement vectorized cursors but we were adding vectorization tests in the earlier code. This check was leading to cases where we were running strategy-v1 with vectorize test parameter as true which led to mismatch in assertions.